config_files/training/config_lorem_ipsum.yaml

```diff
@@ -178,7 +178,25 @@ wrapped_model:
     sharding_strategy: FULL_SHARD
     block_names: [GPT2Block]
 
-model:
+model: 
+  component_key: model
+  variant_key: weight_initialized
+  config:
+    model:
+      instance_key: model_raw
+      pass_type: BY_REFERENCE
+    weight_initializer:
+      component_key: weight_initialization
+      variant_key: composed
+      config:
+        model_type: gpt2
+        weight_init_type: scaled_embed
+        mean: 0.0
+        plain_std: 0.02
+        hidden_dim: ${model_raw.config.n_embd}
+        num_layers: ${model_raw.config.n_layer}
```

Comment:
@flxst, are these the expected references?

Reply:
Yes, looks good.